packages/client/src/store/userSlice.ts

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+interface UserState {
+  isAuth: boolean
+  firstName: string
+  secondName: string
+  displayName: string
+  login: string
+  email: string
+  phone: string
+}
+
+const initialState: UserState = {
+  isAuth: true,
+  firstName: '',
+  secondName: '',
+  displayName: 'Алекс',
+  login: '',
+  email: '',
+  phone: '',
+}
+
+const userSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {
+    setAuthTrue(state) {
+      state.isAuth = true
+    },
+    setAuthFalse(state) {
+      state.isAuth = false
+    },
+    toggleAuth(state) {
+      state.isAuth = !state.isAuth
+    },
+  },
+})
+
+export const { setAuthTrue, setAuthFalse, toggleAuth } = userSlice.actions
+export default userSlice.reducer
```

Comment:
я бы не использовал по дефолту. В разных кмопонентах может быть названо по-разному

Reply:
> я бы не использовал по дефолту. В разных кмопонентах может быть названо по-разному

Я тоже за единообразие. Тем более в секции обсуждений под PR я всякий раз запрашиваю корректировку экспортов по умолчанию на именные экспорты, ссылаясь при этом на сообщение от проверяющего. Однако в данном случае, чтобы избежать экспорта по умолчанию придётся городить вот такую конструкцию:
```suggestion
export const userSlice = createSlice({
  name: 'user',
  initialState,
  reducers: {
    setAuthTrue(state) {
      state.isAuth = true
    },
    setAuthFalse(state) {
      state.isAuth = false
    },
    toggleAuth(state) {
      state.isAuth = !state.isAuth
    },
  },
}).reducer
```